### ISSUE TITLE
Entering interactions where orders are switched in subsequent models generates inconsistent basis set

### ISSUE THREAD

**Issue by jslefche** [jslefche/piecewiseSEM#30]

**Comment by jslefche** [comment]
Going to elevate this because of another complaint. Example:

```
library(MASS)

data = data.frame(
  y = sample(1:100, 100, replace = T),
  x1 = runif(100),
  x2 = runif(100), 
  x3 = runif(100), 
  x4 = runif(100)
)

modelList = list(
  
  glm.nb(y ~ x1*x2, data),
  
  lm(x3 ~ y, data),
  
  lm(x4 ~ x2*x1, data)
  
)

library(piecewiseSEM)

sem.fit(modelList, data)
```

**Comment by jslefche** [comment]
Fixed in 1.0.5
